lib/tsd-file-api-client/build.gradle.kts

```diff
@@ -17,19 +18,35 @@ dependencies {
     annotationProcessor("org.projectlombok:lombok:1.18.34")
 
     implementation("org.apache.commons:commons-lang3:3.17.0")
-    implementation("commons-io:commons-io:2.16.1")
+    implementation("commons-io:commons-io:2.17.0")
     implementation("com.auth0:java-jwt:4.4.0")
     implementation("com.google.code.gson:gson:2.11.0")
 
     api("com.squareup.okhttp3:okhttp:4.12.0")
     implementation("org.slf4j:slf4j-jdk14:2.0.16")
 
-    testImplementation("org.junit.jupiter:junit-jupiter:5.11.0")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.11.1")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 
 tasks.test {
     useJUnitPlatform()
 }
 
-// TODO: Configure the publishing settings for distributing the library/application.
+publishing {
+    publications {
+        create<MavenPublication>("mavenJava") {
+            from(components["java"])
+        }
+    }
+    repositories {
+        maven {
+            name = "fega-norway-tsd-file-api-client"
+            url = uri("https://maven.pkg.github.com/ELIXIR-NO/FEGA-Norway")
+            credentials {
+                username = System.getenv("GITHUB_ACTOR")
+                password = System.getenv("GITHUB_TOKEN")
+            }
+        }
+    }
+}
```

Comment:
Add a newline here :) at the `eof`. I do this mistake all the time as well.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline